drivers/adodb-mysqli.inc.php

```diff
@@ -157,17 +157,67 @@ function SetTransactionMode($transaction_mode)
 	 * Parameter must be one of the constants listed in mysqli_options().
```

Comment:
This needs to be updated to reflect the newly added possible values.

Reply:
Updated PHPdoc with new possible values.